tests/Doctrine/Tests/Common/DataFixtures/Purger/MongoDBPurgerTest.php

```diff
@@ -31,14 +33,10 @@ private function getDocumentManager()
         $config->setHydratorDir($root . '/generate/hydrators');
         $config->setHydratorNamespace('Hydrators');
         $config->setMetadataDriverImpl(AnnotationDriver::create(dirname(__DIR__) . '/TestDocument'));
-        AnnotationDriver::registerAnnotationClasses();
 
-        $dm = DocumentManager::create(null, $config);
-        if (! $dm->getConnection()->connect()) {
-            $this->markTestSkipped('Unable to connect to MongoDB');
```

Comment:
So we never skip the test now?

Reply:
Tried getting this done quickly and removed this. I can add a `ping` command and skip if it fails.

Reply:
Yes please, I see the CI is green, but this is probably targeted at contributors

Reply:
Yep, their tests will fail unless they have a standalone MongoDB instance running.